lib/executor.js

```diff
@@ -20,7 +20,8 @@ class TaskExec extends EventEmitter {
 
     process(triggerOrAction, payload, cfg, snapshot) {
         const onError = async (err) => {
-            this.logger.error(err);
+            this.logger.trace(err);
+            this.logger.error('Error occurred');
```

Comment:
Replace with something more meaningful, like `general error when trying to process a message`.